docs/quick_start_guide.md

```diff
@@ -0,0 +1,63 @@
+# Quick Start Guide
+
+Whether you want to add Kotlin DL into your project, or experiment with it in a Jupyter Notebook, first you'll need to 
+set up appropriate dependencies.  
+
+## Working with Kotlin DL in an IDE
+1. Open a Kotlin project where you want to use Kotlin DL, or create a new Kotlin project in IntelliJ IDEA as described 
+in [Kotlin documentation](https://kotlinlang.org/docs/tutorials/jvm-get-started.html).
+2. Add the Kotlin DL dependency to your project's build file. 
+    * If you're using Gradle as build system, add the following to the `build.gradle.kts` file:
+    ```kotlin
+    repositories {
+        maven(url = "https://kotlin.bintray.com/kotlin-datascience")
+    }
+    
+    dependencies {
+       implementation ("org.jetbrains.kotlin.kotlin-dl:api:[Kotlin DL version]]")
+    }
+    ```  
+   * If your project is built with Maven, add Kotlin DL to your `pom.xml`: 
+   ```xml
+   <repositories>
+       <repository>
+           <id>jcenter</id>
+           <name>jcenter</name>
+           <url>https://kotlin.bintray.com/kotlin-datascience</url>
+       </repository>
+   </repositories>
+   
+   <dependency>
+     <groupId>org.jetbrains.kotlin.kotlin-dl</groupId>
+     <artifactId>api</artifactId>
+     <version>[Kotlin DL version]</version>
+     <type>pom</type>
+   </dependency>
+   ```
+ 
+You can also take advantage of Kotlin DL functionality in any existing Java project, even if you have no other Kotlin 
+code in it yet. Check out [these instructions](https://kotlinlang.org/docs/tutorials/mixing-java-kotlin-intellij.html#adding-kotlin-source-code-to-an-existing-java-project) 
+on how to add Kotlin code  in your existing Java code base. 
```

Comment:
I have an example of simple LeNet in Java https://github.com/JetBrains/KotlinDL/blob/release_0.1/examples/src/main/java/LeNetClassic.java
maybe it will be useful to add

Reply:
That's not what I meant by adding Kotlin code to an existing Java code base. I mean having .kt files in a Java code base.